plugins/woocommerce/client/admin/client/settings-payments/onboarding/providers/woopayments/steps/business-verification/index.tsx

```diff
@@ -30,10 +30,7 @@ export const BusinessVerificationStep: React.FC = () => {
 			location.hostname === 'localhost'
 				? 'https://wcpay.test'
 				: window.wcSettings?.homeUrl + getComingSoonShareKey(),
-		country: (
-			window.wcSettings?.admin?.preloadSettings?.general
-				?.woocommerce_default_country || 'US'
-		).split( ':' )[ 0 ],
```

Comment:
I'm wondering if there are some edge cases here, and we should keep the split just to be safe 🤔

Reply:
Added this back in https://github.com/woocommerce/woocommerce/pull/58190/commits/656118ae36274193d538ad924bf84af8b9f972d1

Reply:
The country switcher component only supports two-digit country codes. In my view, the split isn’t necessary.

Reply:
Good call, reverted in 53ae88d736ccca300dc817f4640ecceeaaf63879